package.json

```diff
@@ -27,6 +27,7 @@
     "dayjs": "^1.8.8",
     "echarts": "^5.0.2",
     "lodash": "^4.17.15",
+    "monaco-editor": "^0.27.0",
```

Comment:
Remember to add the license when updating the submodule

Reply:
> There is no error message if the backend did not enable this feature, otherwise LGTM

![3](https://user-images.githubusercontent.com/20871783/132185502-165fc947-687a-4309-a6a8-79dedf7b0c5d.png)
Done